Automattic/wp-calypso
# Rename `wp-calypso-client` to `calypso`

### Background

See #44602

### Changes proposed in this Pull Request

There was a few discussions in #44602 about the name `wp-calypso-client`. This PR renames it to `calypso`. The root package is still named `wp-calypso` (to match the repo).

### Testing instructions

* Verify that test passes (ignore lint errors!)
* Verify that calypso.live loads in Chrome and IE11
* Verify there are no huge differences in bundle size
